### PR TITLE
Repurpose email field to telegram handle field

### PR DESCRIPTION
**Pull request by WR3nd3**
prefix for telegram handle is tele/

### CODE REVIEW

File: src/main/java/paymelah/model/person/Address.java [hunk 1]
```diff
@@ -31,7 +31,7 @@ public Address(String address) {
     }
 
     /**
-     * Returns true if a given string is a valid email.
+     * Returns true if a given string is a valid telegram handle.
```
Comment (lyuting47) [inline src/main/java/paymelah/model/person/Address.java:34]: This is for the physical `Address` not email address. So should be "Returns true if a given string is a valid address"

**Review by lyuting47 — CHANGES_REQUESTED** [review]

**Review by lyuting47 — APPROVED** [review]

File: docs/UserGuide.md [hunk 2]
```diff
@@ -253,11 +253,11 @@ If your changes to the data file makes its format invalid, PayMeLah will discard
 
 Action | Format, Examples
 --------|------------------
-**Add person** | `add n/<name> p/<phone number> e/<email> a/<address> [t/<tag>]…` <br> e.g., `add n/James Ho p/22224444 e/jamesho@example.com a/123, Clementi Rd, 1234665 t/friend t/colleague`
+**Add person** | `add n/<name> p/<phone number> tele/<teelgram> a/<address> [t/<tag>]…` <br> e.g., `add n/James Ho p/22224444 tele/James_H0 a/123, Clementi Rd, 1234665 t/friend t/colleague`
```
Comment (waynezsy) [inline docs/UserGuide.md:256]:
```suggestion
**Add person** | `add n/<name> p/<phone number> tele/<telegram> a/<address> [t/<tag>]…` <br> e.g., `add n/James Ho p/22224444 tele/James_H0 a/123, Clementi Rd, 1234665 t/friend t/colleague`
```

**Review by waynezsy — CHANGES_REQUESTED** [review]

File: docs/UserGuide.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (WR3nd3) [reply docs/UserGuide.md]: Resolved

**Review by waynezsy — DISMISSED** [review]
LGTM

**Pull request merged**